SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -230,11 +230,17 @@ public SingularityRequest checkSingularityRequest(
       !request.getInstances().isPresent() || request.getInstances().get() > 0,
       "Instances must be greater than 0"
     );
+
+    // check if requested number of instances exceeds max scale
+    int maxInstances = request.getMaxScale().isPresent()
+      ? request.getMaxScale().get()
+      : maxInstancesPerRequest;
+
     checkBadRequest(
-      request.getInstancesSafe() <= maxInstancesPerRequest,
-      "Instances (%s) be greater than %s (maxInstancesPerRequest in mesos configuration)",
+      request.getInstancesSafe() <= maxInstances,
+      "Instances (%s) cannot be greater than %s",
```

Comment:
Could be helpful to users to state which configuration value they're being blocked by here